docs/src/pages/[platform]/connected-components/authenticator/configuration/socialProviders.react-native.mdx

```diff
@@ -0,0 +1,20 @@
+import { Message } from '@aws-amplify/ui-react';
+import { Example } from '@/components/Example';
+import { Fragment } from '@/components/Fragment';
+import {SocialProviderDemo} from "@/components/SocialProviderDemo";
+
+## Social Providers
+
+<Message colorTheme="info" heading="Zero Configuration">
+
+The Authenticator [automatically infers](/connected-components/authenticator#step-1-configure-backend) `socialProviders` from `amplify pull`,
+but can be explicitly defined as seen below.
```

Comment:
Prefer "reads" over "infers" as the configuration itself is explicit. IMO the `socialProviders` override is more likely to cause confusion than provide much of a use case since the oauth configuration has to exist in the config file